www/include/options/media/images/DB-Func.php

```diff
@@ -430,53 +430,70 @@ function getListDirectory($filter = null)
  */
 function isCorrectMIMEType(array $file): bool
 {
+    $mimeTypeFileExtensionConcordance = [
+        "svg" => "image/svg+xml",
+        "jpg" => "image/jpeg",
+        "jpeg" => "image/jpeg",
+        "gif" => "image/gif",
+        "png" => "image/png"
+    ];
+    $fileExtension = end(explode(".", $file["name"]));
     $mimeType = mime_content_type($file['tmp_name']);
 
-    if (!preg_match('/(^image\/)|(^application(\/zip)|(\/x-gzip)$)/', $mimeType)) {
+    if (
+        !preg_match('/(^image\/(jpg|jpeg|svg\+xml|gif|png)$)|(^application(\/zip)|(\/x-gzip)$)/', $mimeType)
+        || (preg_match('/^image\//', $mimeType) && $mimeType !== $mimeTypeFileExtensionConcordance[$fileExtension])
```

Comment:
`$mimeTypeFileExtensionConcordance[$fileExtension]` will generate a NOTICE message if the content of `$fileExtension` is not listed in `$mimeTypeFileExtensionConcordance`